pvlib/iotools/tmy.py

```diff
@@ -3,9 +3,25 @@
 import datetime
 import re
 import pandas as pd
-
-
-def read_tmy3(filename, coerce_year=None, recolumn=True):
+import warnings
+from pvlib._deprecation import pvlibDeprecationWarning
+
+# Dictionary mapping TMY3 names to pvlib names
+VARIABLE_MAP = {
```

Comment:
Should `ERTN` and `Pwat` be mapped as well?

Reply:
Good catch!

"ETRN (W/m^2)" I believe should be mapped to `dni_extra`, "ETR (W/m^2)" should be mapped to `ghi_extra`, and "Pwat (cm)" to `precipitable_water`.